server/routes/creatingLicences/index.ts

```diff
@@ -39,7 +43,7 @@ export default function Index({ licenceService, caseloadService }: Services): Ro
   const get = (path: string, handler: RequestHandler) =>
     router.get(routePrefix(path), fetchLicence(licenceService), asyncMiddleware(handler))
   const post = (path: string, handler: RequestHandler, type?: new () => unknown) =>
-    router.post(routePrefix(path), validationMiddleware(type), asyncMiddleware(handler))
+    router.post(routePrefix(path), fetchLicence(licenceService), validationMiddleware(type), asyncMiddleware(handler))
 
```

Comment:
For the 1st POST in the form journey there won't be a licence. Will this be an issue for fetchLicence on all posts?

Reply:
For the first POST there wont be a licenceId already, and so the fetch licence middleware will get skipped over!